onnxscript/rewriter/_ir_utils.py

```diff
@@ -47,7 +47,5 @@ def propagate_const_value(ir_value: ir.Value) -> ir.Value:
 def get_numpy_from_ir_value(value: ir.Value) -> np.ndarray | None:
     constant_value = value.const_value
     if constant_value is not None:
-        if isinstance(constant_value, ir.serde.TensorProtoTensor):
-            return constant_value.numpy()
-        return np.array(constant_value)
+        return constant_value.numpy()
     return constant_value
```

Comment:
Instead of this, I suggest removing the function and move the None check before the call site, since we have to check for None anyways

Reply:
Before we call const_value?

Reply:
Yes. So

if value.const_value is None:
    # do something
else:
    something_else = value.const_value.numpy()

Reply:
However if we feat the `numpy()` call may fail, we can create a 

```
def try_get_numpy(value: ir.Value) -> np.ndarray:
    try:
        ...
    except ...:
        return None
```

Reply:
But it doesn't feel very elegant :/